util/install.sh

```diff
@@ -397,7 +405,8 @@ function ovs {
     echo "Installing Open vSwitch..."
 
     if [ "$DIST" = "Fedora" -o "$DIST" = "RedHatEnterpriseServer" ]; then
-        $install openvswitch openvswitch-controller
+        $install openvswitch
```

Comment:
Older RHEL needed this apparently?

Reply:
Probably, but I am not sure. Giving this problem a proper solution may require more effort than I thought. The Fedora package generated directly from ovenvswitch sources has the controller, but the official package included in Fedora doesn't. I will see what can be done to include the controller in the Fedora package. Installing openvswitch from the source is the only way to go until that part is solved.
https://bugzilla.redhat.com/show_bug.cgi?id=1914605

Once that part is solved, I might create a Fedora package for mininet myself.